src/shared_utilities/utils.jl

```diff
@@ -9,7 +9,11 @@ export FTfromY
 Computes the heaviside function.
 """
 function heaviside(x::FT)::FT where {FT}
-    if x > eps(FT)
+    heaviside(x, FT(0))
+end
```

Comment:
this function could also be shortened into one line:
```
    heaviside(x::FT)::FT where {FT} = heaviside(x, FT(0))
```

Reply:
thanks!